### ISSUE TITLE
No push notification with VPN service WireGuard in Android 9

### ISSUE THREAD

**Issue by armhub** [tutao/tutanota#999]

**Comment by charlag** [comment]
@armhub is this still the case?

**Comment by ghost** [comment]
> @armhub is this still the case?

yeah sadly it still is got two emails and no push while without vpn it came instantly and sometimes faster than googles push but now nothing :(

- android 16
- using always on vpn and block any connection without vpn turning both off has no effect 
- turning vpn off let's a delayed push come through from emails i already read but only more recent few minute old ones not hours old


Edit 1 Day later:
no change in vpn, tuta app or any settings and it somehow got fixed only thing different is a update to my phone will look if it reverts back or stays like this.

Edit 2 Days later: It's back not sending out pushes when a email comes in.. 

Edit 5 days later: Sometimes it works most often it's not working.

**Comment by ghost** [comment]
found one mention of this though not directly from the author but people responding to this issue.

https://github.com/tutao/tutanota/issues/6067
